webclient/src/types/languages.ts

```diff
@@ -0,0 +1,11 @@
+import { CountryLabel } from './countries';
+
+export enum Language {
+  'en-US' = 'en-US',
+  'es' = 'es',
+}
+
+export enum LanguageCountry {
```

Comment:
You are entirely correct, i chose a spanish speaking country at random, so that i could point to one of our flags for the dropdown images.

Everything else you said brings up good questions, and ones id like to bring up with @ebbit1q and @ZeldaZach on the next call.

Reply:
Copy that.  Figuratively holding my horses.